src/lib.rs

```diff
@@ -23,6 +23,7 @@ use std::result::Result;
 use std::sync::Once;
 #[macro_use]
 extern crate bitfield;
+pub mod ffi;
```

Comment:
So the bindings you generate aren't actually used yet though?

Reply:
Currently they are not used, to avoid merge conflicts. It will replace manually wrote bindings in future PRs.